doc/classes/TileMapLayer.xml

```diff
@@ -0,0 +1,309 @@
+			</description>
+		</method>
+		<method name="map_pattern">
+			<return type="Vector2i" />
+			<param index="0" name="position_in_tilemap" type="Vector2i" />
+			<param index="1" name="coords_in_pattern" type="Vector2i" />
+			<param index="2" name="pattern" type="TileMapPattern" />
+			<description>
+				Returns for the given coordinate [param coords_in_pattern] in a [TileMapPattern] the corresponding cell coordinates if the pattern was pasted at the [param position_in_tilemap] coordinates (see [method set_pattern]). This mapping is required as in half-offset tile shapes, the mapping might not work by calculating [code]position_in_tile_map + coords_in_pattern[/code].
+			</description>
+		</method>
+		<method name="map_to_local" qualifiers="const">
+			<return type="Vector2" />
+			<param index="0" name="map_position" type="Vector2i" />
+			<description>
+				Returns the centered position of a cell in the TileMapLayer's local coordinate space. To convert the returned value into global coordinates, use [method Node2D.to_global]. See also [method local_to_map].
+				[b]Note:[/b] This may not correspond to the visual position of the tile, i.e. it ignores the [member TileData.texture_origin] property of individual tiles.
+			</description>
+		</method>
+		<method name="notify_runtime_tile_data_update">
+			<return type="void" />
+			<description>
+				Notifies the TileMapLayer node that calls to [method _use_tile_data_runtime_update] or [method _tile_data_runtime_update] will lead to different results. This will thus trigger a TileMapLayer update.
+				[b]Warning:[/b] Updating the TileMapLayer is computationally expensive and may impact performance. Try to limit the number of calls to this function to avoid unnecessary update.
+				[b]Note:[/b] This does not trigger a direct update of the TileMapLayer, the update will be done at the end of the frame as usual (unless you call [method update_internals]).
+			</description>
+		</method>
+		<method name="set_cell">
+			<return type="void" />
+			<param index="0" name="coords" type="Vector2i" />
+			<param index="1" name="source_id" type="int" default="-1" />
+			<param index="2" name="atlas_coords" type="Vector2i" default="Vector2i(-1, -1)" />
+			<param index="3" name="alternative_tile" type="int" default="0" />
+			<description>
+				Sets the tile identifiers for the cell at coordinates [param coords]. Each tile of the [TileSet] is identified using three parts:
+				- The source identifier [param source_id] identifies a [TileSetSource] identifier. See [method TileSet.set_source_id],
+				- The atlas coordinates identifier [param atlas_coords] identifies a tile coordinates in the atlas (if the source is a [TileSetAtlasSource]). For [TileSetScenesCollectionSource] it should always be [code]Vector2i(0, 0)[/code],
```

Comment:
```suggestion
				- The atlas coordinate identifier [param atlas_coords] identifies a tile coordinate in the atlas (if the source is a [TileSetAtlasSource]). For [TileSetScenesCollectionSource] it should always be [code]Vector2i(0, 0)[/code],
```

Reply:
I don't think this is right. A Vector2 has two coordinates, so it should be plural.

Reply:
See [here](https://en.wikipedia.org/wiki/Noun_adjunct)

Reply:
>  i.e. when it modifies "identifier"

Hmm. I don't think it modifies the "identifier". It's more of an "identifier made of two coordinates".

Reply:
Alright. I don't mind.